api/http/templates.go

```diff
@@ -74,229 +74,1311 @@ const bzzList = `{{ define "content" }}
4t8K9BG2t8hK+zlfHVbj/Fu4tcG+BewvcW+DeAvcWuG+FexjuYbiH4R6GQRjup+HeCve34H6IfdVETnRe8qjCUmnCm2bi5OOZYjfkMp+gDmeQuYL0dnbgTuDgHLvC3CRzSdkH9HwKHx8yJ2AtK16QrfAuRq8lu3g2MPYydbaX3XAKmX4wAAaJ4RnmzoExMI7WKM8U+6yATmaZTrZLHqNKH6NrWXQt656uZdG1LLj5yEKK6OdIldlT+sx2o7vQ1PwcvGLwihHPi3CJsdNu0jFy4BKDSwwuMbjE4BKDSwwuMTpDjo4RPHRE7OwVYoSdZmrARx056ElDZgc1wbjZ9PJ5FotMt9/LmTQfpVmiNMvstIlSjonffEy9sNc9eh1sD8B2JlvV16iOScPai77nYa3ZvyCb8GBWtpMlXdE7qOKdPHeBevCiqfIYnFzcNy4SMxdRLyTqz2DFhS9bybBH9vHu4/crRCuMvI5C3wOR+IDcn85GZAiMZCMzmo3OOP0qyjmaon7wCCsp9ocb7l/APcGZVwr/BLwn4euE1yR2EthIYCOBjQS6J9E5ib5JVn/FTVrX3TJisZM4HEDXNLrOGz1ebjq1PLcxtsv4mSCLX8M/YfT2gfcf0H/a2EjAPwH/BLYS2EqYKtNW9sPahaVDWJrB0kVTv15zc0uaSO/S5yi/w0DXbR94/776TWIliZUkVpJYSWIliZWkOcscaJxB4/yu0LtgN9AVr+1vw7b/Hi/jSF5CMo7tqayXcby8jf049uPYj2M/jv049uPYj2M/jsY49uPYj2M/jv049hP6Ho+m+XM8xLk5zYl26IF7X5u+M7Mfvu0td8V3di/Wd1997/3/LX5baws37dLv1McFqzaTrdy79y5938r7xk1Mjzr+6/1M6xg1/3Uf+l9nvvO+2ecYeVAi974zQZ8HV0zP9xmWCyeA7mS5dzud7nI20/vy/mMH1BXpgfmn3Ixc3IxKuRnZqUgPFemhIj3ScadPf6ByT57/jignnkeJ4zFO7yi74RJ6J9A7nT2bplldzupyVpezutN84WzBxgw2HNhYhsQWJLYgsQWJk0aiAombzFrE7RYSFUhUIFGBxLs6wvCr4ru0Wm0iwr9Daiu75cbdPTV0z54KfGNP5XD3dFM9wqm4gviUyVPcwcq5zxbxhbyO0QpZzw3+B7IFfh5Ogsc56+rkSTq6D6mD/FXLYb4JarjTdsqzfM/1yA/5iuMOKWFu8ds5Dwc5A87IEJkbkVFu/GP87ZGkpOj7f5d/0PUz/O37N9wK67kAAAB42mNgZGBg4GJQYLBiYMxJLMlj4GBgAYow/P/PwASkGBmY0ooSkxn4cjLTExlEwCKMYJIBKM/GwAdWzcggAKU1gFgKiDnAsjwMz4G0P8MzIOkD1ukJxDpAHshGiP56BmYGIQZRhjowLQ6UZ2FogLOZgHaKAQAj+w1qAAAAeNpjYGR+xDiBgZWBhXEW4ywGBhAJoRlSmAQYkEADA5N+AIOCF4wfnJibz+DAwPubiS3tXxoDA4cQkwhQePr5awwMADRDDhoAeNpjYGBgZoBgGQZGBhC4AuQxgvksDDuAtBaDApDFxcDLUMfwnzGYsYLpGNMdBS4FEQUpBTkFJQU1BX0FK4V4hTWKSqp/fjP9/w/UwwvUs4AxCKqWQUFAQUJBBqrWEq6W8f///1//P/5/6H/Bf5+///++enD8waEH+x/se7D7wY4HGx4sf9D8wPz+oVsvWZ9C3UYkYGRjgGtgZAISTOgKgF5mYWVj5+Dk4ubh5eMXEBQSFhEVE5eQlJKWkZWTV1BUUlZRVVPX0NTS1tHV0zcwNDI2MTUzt7C0sraxtbN3cHRydnF1c/fw9PL28fXzDwgMCg4JDQuPiIyKjomNi09IZGhr7+yePGPe4kVLli1dvnL1qjVr16/bsHHz1i3bdmzfs3vvPoailNTMuxULC7KflGUxdMxiKGZgSC8Huy6nhmHFrsbkPBA7t/ZeUlPr9EOHr167dfv6jZ0MB48wPH7w8Nlzhsqbdxhaepp7u/onTOybOo1hypy5sxmOHisEaqoCYgA0MoqeAAAAAARgBekAiQCKAIsAjACRAJYBAgCXAKIAlwCYAJkAmgCcAJ4AlQCCAIcAjwCTAEQFEQAAeNpdUbtOW0EQ3Q0PA4HE2CA52hSzmZDGe6EFCcTVjWJkO4XlCGk3cpGLcQEfQIFEDdqvGaChpEibBiEXSHxCPiESM2uIojQ7O7NzzpkzS8qRqnfpa89T5ySQwt0GzTb9Tki1swD3pOvrjYy0gwdabGb0ynX7/gsGm9GUO2oA5T1vKQ8ZTTuBWrSn/tH8Cob7/B/zOxi0NNP01DoJ6SEE5ptxS4PvGc26yw/6gtXhYjAwpJim4i4/plL+tzTnasuwtZHRvIMzEfnJNEBTa20Emv7UIdXzcRRLkMumsTaYmLL+JBPBhcl0VVO1zPjawV2ys+hggyrNgQfYw1Z5DB4ODyYU0rckyiwNEfZiq8QIEZMcCjnl3Mn+pED5SBLGvElKO+OGtQbGkdfAoDZPs/88m01tbx3C+FkcwXe/GUs6+MiG2hgRYjtiKYAJREJGVfmGGs+9LAbkUvvPQJSA5fGPf50ItO7YRDyXtXUOMVYIen7b3PLLirtWuc6LQndvqmqo0inN+17OvscDnh4Lw0FjwZvP+/5Kgfo8LK40aA4EQ3o3ev+iteqIq7wXPrIn07+xWgAAAAABAAH//wAPeNrNvQ18E+eVLzzPzGj0YUnW6MPyty0LWxhhhCVkoRhjYhziEpf6ul6Xui5xqEMcJ4QQSqjX9VJf6roOJZTSYMVxWcLycl1f1r+RrBBKCIGkNJtlWTaXC2yW5celLMt1k6bZbG42JfbwnvPMSP7ABLLv/va+IbZHI2nmOec5zzn/c55zzjAsU80wbKvmTxiO0TILooTxLYlpeefv/VFB849LYhwLh0yUw9MaPB3TCunjS2IEzwdEl1joEl3VbL48h0TkNs2f3Pz
```

Comment:
I did not expect that clicking on the hash would copy it. There already is a button for that on the right. Maybe someone would not want to replace the content of the clipboard just by clicking on text.